src/QA/QA.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -37,22 +37,30 @@ public async Task<DashboardViewModel> GetDashboardViewModelAsync(string searchTe
             if (vm.DisplayInProgressVacancies)
             {
                 var inProgressSummaries = await _vacancyClient.GetVacancyReviewsInProgressAsync();
-                vm.InProgressVacancies = inProgressSummaries.Select(v => MapToViewModel(v, vacancyUser)).ToList();
+
+                var inProgressVacanciesVmTasks = inProgressSummaries.Select(async v => await MapToViewModelAsync(v, vacancyUser)).ToList();
```

Comment:
We don't need the async/await here as we're not doing anything with the result inside the lambda. So we can change it so it looks like the one on line 51.

Reply:
Made the changes.